src/StoreKit/NativeMethods.cs

```diff
@@ -5,7 +5,7 @@
 namespace StoreKit {
 
 	partial class SKReceiptRefreshRequest {
-		[iOS (7,1), Mac (10,14)]
+		[Watch (6,2), iOS (7,1), Mac (10,14)]
```

Comment:
probably not needed
since StoreKit is new in watchOS 6.2 then you only have to decorate all (available) types with the attributes
IOW you don't need to add them on members